### PR TITLE
Accessibility improvements

### PR DESCRIPTION
**Pull request by lsfgrd**
## Post list
- Removed the post image alt text that was just repeating the post title. In this case, I consider the image to be just decorative. It is better to have an empty alt text (screenreaders would just read the post title twice).
- Removed extra empty `<p>` elements being added in the post excerpt (Hugo already handles the paragraphs).

## Footer
Changed footer text color to be compliant with minimum contrast requirements:
image

### CODE REVIEW

File: layouts/partials/list.html [hunk 1]
```diff
@@ -14,13 +14,13 @@
     <article class="card mb-3">
       <div class="row g-0">
         <div class="col-md-4">
-          {{ partial "helpers/image" (dict "filename" (printf "%s%s" .Params.image "?width=400px") "alt" .Title "class" "img-fluid rounded-start" "resources" .Resources) }}
+          {{ partial "helpers/image" (dict "filename" (printf "%s%s" .Params.image "?width=400px") "alt" "" "class" "img-fluid rounded-start" "resources" .Resources) }}
```
Comment (elyezer) [inline layouts/partials/list.html:17]: Wound't be better for accessibility if we had the alternate text so screen readers could read it?

File: layouts/partials/list.html [hunk 2]
```diff
@@ -14,13 +14,13 @@
...
         </div>
         <div class="col-md-8">
           <div class="card-body">
             <h2 class="card-title post-title"><a href="{{ .Permalink }}">{{ partial "helpers/title" . }}</a></h2>
             {{- partial "post/meta" . -}}
-            <p class="card-text post-summary">{{ partial "post/excerpt" . }}</p>
+            {{ partial "post/excerpt" . }}
```
Comment (elyezer) [inline layouts/partials/list.html:23]: Please replace with a div then, so that the classes and styling could still be applied.

**Review by elyezer — CHANGES_REQUESTED** [review]

File: layouts/partials/list.html (hunk 1 above)
Comment (lsfgrd) [reply layouts/partials/list.html:17]: @elyezer the reasoning here is that currently the alt text is just the post title, which is already available for screenreaders to read it

By replacing it with an empty alt, the image is considered decorative and is skipped, so the post title is not read twice

File: layouts/partials/list.html (hunk 2 above)
Comment (lsfgrd) [reply layouts/partials/list.html:23]: @elyezer I double checked and it seems like the styles are doing nothing 😅 The classes do not exist, and actually this is the structure being currently rendered on the live site:
image

File: layouts/partials/list.html (hunk 1 above)
Comment (elyezer) [reply layouts/partials/list.html:17]: This is a great suggestion, thank you!

File: layouts/partials/list.html (hunk 2 above)
Comment (elyezer) [reply layouts/partials/list.html:23]: Something is happening with the rendering, It was not supposed to be empty, see here https://github.com/CastalioPodcast/castaliopodcast.github.io/blob/main/layouts/partials/list.html#L23

The card-text comes from the bootstrap css framework and checking there there is no special styling and the post-summary was inherited from the site when it was using pelican. With that we can go ahead and merge this one. Later I will need to improve the list template as recently I moved all content to use markdown and the excerpt needs some fixing.

**Review by elyezer — APPROVED** [review]

**Comment by elyezer** [comment]
Thank you for helping improving the podcast site.

**Pull request merged**